### PR TITLE
feat: create fastapi service

### LINKED ISSUE #43: Create FastAPI service
This tool is used in quite a few of our services ([variation-normalization](https://github.com/cancervariants/variation-normalization), [fusor](https://github.com/cancervariants/fusor), and [fusion-curation](https://github.com/cancervariants/fusion-curation)) and I think it has functionality that others would be interested in using, so we should make this a FastAPI service

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #43 

- Creates endpoints for current methods: `uta_tools/genomic_to_transcript_exon_coordinates` and `uta_tools/transcript_to_genomic_coordinates`

### CODE REVIEW

File: uta_tools/schemas.py [hunk 1]
```diff
@@ -2,28 +2,116 @@
...
+
+
+class TranscriptRequestBody(BaseModelForbidExtra):
+    """Define constraints for transcript exon to genomic coordinates request body"""
+
+    gene: Optional[StrictStr] = None
+    transcript: Optional[StrictStr] = None
+    exon_start: Optional[StrictInt] = None
+    exon_start_offset: Optional[StrictInt] = 0
+    exon_end_offset: Optional[StrictInt] = 0
```
Comment (jsstevenson) [inline uta_tools/schemas.py:79]: duplicate of line 81

File: uta_tools/main.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        genomic_data=None, warnings=list(), service_meta=uta_tools.service_meta())
+
+    try:
+        response = await uta_tools.transcript_to_genomic_coordinates(**request_body)
+    except TypeError as e:
+        error = str(e)
+        logger.error(f"transcript_to_genomic_coordinates TypeError: {error}")
+        response.warnings.append(error)
+    except Exception as e:
+        logger.error(f"transcript_to_genomic_coordinates unhandled exception {str(e)}")  # noqa: E501
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply uta_tools/main.py:110]: Ah copy/paste mistake

File: uta_tools/schemas.py (hunk 1 above)
Comment (korikuzma) [reply uta_tools/schemas.py:79]: Good catch!

**Review by jsstevenson — APPROVED** [review]

File: uta_tools/main.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (korikuzma) [reply uta_tools/main.py]: This is outdated

**Pull request merged**